packer/builder/azure/arm/artifact.go

```diff
@@ -3,34 +3,116 @@
 
 package arm
 
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
 const (
 	BuilderId = "Azure.ResourceManagement.VMImage"
 )
 
-type artifact struct {
-	name string
+type Artifact struct {
+	StorageAccountLocation      string
+	StorageAccountName          string
+	StorageAccountContainer     string
+	StorageAccountBlob          string
+	Template                    string
```

Comment:
My bad.  I should have removed that.  I am not going to put the whole template in there.  The user can access the template via the template URI.